velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -813,5 +819,43 @@ TEST_F(StringTest, replace) {
       replace("123\u6570\u6570\u636E", "\u6570\u636E", "data"),
       "123\u6570data");
 }
+
+TEST_F(StringTest, findInSet) {
+  EXPECT_EQ(findInSet("ab", "abc,b,ab,c,def"), 3);
+  EXPECT_EQ(findInSet("abc", "abc,b,ab,c,def"), 1);
+  EXPECT_EQ(findInSet("ab", "abc,b,ab,ab,ab"), 3);
+  EXPECT_EQ(findInSet("abc", "abc,abc,abc,abc,abc"), 1);
+  EXPECT_EQ(findInSet("c", "abc,b,ab,c,def"), 4);
+  EXPECT_EQ(findInSet("dfg", "abc,b,ab,c,def"), 0);
+  EXPECT_EQ(findInSet("dfg", "dfgdsiaq"), 0);
+  EXPECT_EQ(findInSet("dfg", "dfgdsiaq, dshadad"), 0);
+  EXPECT_EQ(findInSet("", ""), 1);
+  EXPECT_EQ(findInSet("", "123"), 0);
+  EXPECT_EQ(findInSet("123", ""), 0);
+  EXPECT_EQ(findInSet("", "123,"), 2);
+  EXPECT_EQ(findInSet("", ",123"), 1);
+  EXPECT_EQ(findInSet("dfg", std::nullopt), std::nullopt);
+  EXPECT_EQ(findInSet(std::nullopt, "abc"), std::nullopt);
+  EXPECT_EQ(findInSet(std::nullopt, std::nullopt), std::nullopt);
+  EXPECT_EQ(
+      findInSet(
+          "\u0061\u0062",
+          "\u0061\u0062\u0063\u002c\u0062\u002c\u0061\u0062\u002c\u0063\u002c\u0064\u0065\u0066"),
```

Comment:
Since those are all ascii characters. Let's directly use the corresponding character instead of code point for readability. Ditto for other applicable places.